### DISCUSSION TITLE
Bypass always incognito for specific websites

### DISCUSSION THREAD

**Discussion by bigdiff** [bromite/bromite#2177] [Ideas]
I have a couple of web services I use which take a really long time to load because of webfonts and big wasm/js bundles. So being able to use the cache in those cases saves me a good 5 to 10 seconds if opened in the normal mode.

**Comment by uazo** [comment]
it is not possible to bypass the always incognito.
maybe you could try to make your own userscript that injects static resources, even if you would then have the problem of keeping it updated.
